client/verta/verta/client.py

```diff
@@ -1735,3 +1738,23 @@ def find_datasets(
             datasets = datasets.find(predicates)
 
         return datasets
+
+    def _create_workspace(self, org_id, workspace_name, resource_action_groups):
```

Comment:
Doc string?

Reply:
Agreed—this method definitely needs a docstring to explain what it is doing, and what each of the arguments arts.

For example, I'm not sure what exactly `resource_action_groups` is, or how it will be used.